src/Image/index.tsx

```diff
@@ -3,4 +3,13 @@ import { Image, ImageProps } from './Image';
 
 export { Image };
 export type { ImageProps };
+/**
+ * **React Native Elements Image Component**
+ *
+ * **Documentation** https://reactnativeelements.com/docs/image
+ *
+ * **Preview**
+ *
+ * ![Image](https://raw.githubusercontent.com/arpitBhalla/rne.snippets/next/img/rneImage.png)
+ */
```

Comment:
![Screenshot from 2021-06-24 17-07-25](https://user-images.githubusercontent.com/55053424/123261400-59f19980-d514-11eb-98f6-42847b583be7.png)

Reply:
https://github.com/react-native-elements/react-native-elements/pull/3112 Please follow the pattern as this, I have discussed today with mentors. Any change related to preview or usage would be done once scripts are written and auto-gen is working as a new feature, not now.

Reply:
Okay 😒